Userland/Libraries/LibWeb/WebIDL/ExceptionOr.h

```diff
@@ -59,9 +60,18 @@ class ExceptionOr {
     {
     }
 
-    ExceptionOr(Variant<SimpleException, JS::NonnullGCPtr<DOMException>> exception)
-        : m_exception(move(exception).template downcast<Empty, SimpleException, JS::NonnullGCPtr<DOMException>>())
+    ExceptionOr(JS::Completion exception)
+        : m_exception(move(exception))
+    {
+        auto const& completion = m_exception.get<JS::Completion>();
+        VERIFY(completion.is_error());
+    }
```

Comment:
This constructor seems useless, JS::Completion should already be accepted by the one below